lib/client.js

```diff
@@ -367,25 +368,24 @@ extend(Raven.prototype, {
       cb = kwargs;
       kwargs = {};
     } else {
-      kwargs = kwargs || {};
+      // Do not use reference, as we'll modify this object later on
+      kwargs = kwargs ? JSON.parse(stringify(kwargs)) : {};
```

Comment:
I read through the codebase once again and it appears that we should be perfectly fine.

https://github.com/getsentry/raven-node/blob/master/lib/client.js#L549-L562

This middleware is passing just an `err` itself, so we only have to preserve `req` just in case someone wants to use it inside `dataCallback`.